src/plugins/cosmos/components/modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -27,29 +29,21 @@ export const useFormSend = () => {
           .times(bn(10).exponentiatedBy(data.asset.precision))
           .toFixed(0)
 
-        const adapterType = adapter.getType()
-
         let result
 
         const { memo, estimatedFees, feeType, address: to } = data
-        if (adapterType === ChainTypes.Cosmos) {
-          const fees = estimatedFees[feeType] as chainAdapters.FeeData<ChainTypes.Cosmos>
-          const gas = fees.chainSpecific.gasLimit
-          const fee = fees.txFee
-          const address = to
-          result = await (adapter as ChainAdapter<ChainTypes.Cosmos>).buildSendTransaction({
-            to: address,
-            memo,
-            value,
-            wallet,
-            chainSpecific: { gas, fee },
-            sendMax: data.sendMax,
-          })
-        } else if (adapterType === ChainTypes.Osmosis) {
-          // TODO(gomes): implement this
-        } else {
-          throw new Error('unsupported adapterType')
-        }
+        const fees = estimatedFees[feeType] as CosmosSdkChainFees
```

Comment:
export type SendInput = {
  [SendFormFields.Address]: string
  [SendFormFields.Memo]?: string
  [SendFormFields.AccountId]: AccountSpecifier
  [SendFormFields.AmountFieldError]: string | [string, { asset: string }]
  [SendFormFields.Asset]: Asset
  [SendFormFields.FeeType]: chainAdapters.FeeDataKey
  [SendFormFields.EstimatedFees]: chainAdapters.FeeDataEstimate<ChainTypes>
  [SendFormFields.CryptoAmount]: string
  [SendFormFields.CryptoSymbol]: string
  [SendFormFields.FiatAmount]: string
  [SendFormFields.FiatSymbol]: string
  [SendFormFields.SendMax]: boolean
}

because of fees

Reply:
SendInput has feeData which is chain specific